publisher/socials/vk.py

```
@@ -0,0 +1,98 @@
+        except Exception as e:
+            return self.error(f"Ошибка авторизации: {e}")
+     
+        vk = vk_session.get_api()
+        
+        content, photo_src = self.clean_publish_data(post.content)
+        
+        if (photo_src):
+            # Скачиваем изображение, если это URL
+            if photo_src.startswith('http'):
+                try:
+                    tmp = tempfile.NamedTemporaryFile(delete=False)
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)